Cargo.toml

```
@@ -22,7 +22,8 @@ use-serde = ["serde", "bitcoin_hashes/serde"]
 [dependencies]
 bech32 = "0.7.1"
 byteorder = "1.2"
-bitcoin_hashes = "0.7"
+#bitcoin_hashes = "0.7"
+bitcoin_hashes = { git = "https://github.com/stevenroose/bitcoin_hashes.git", branch = "sha256t" }
```

With the PR accepted, we now can change this back to the original crate

```suggestion
bitcoin_hashes = "0.9.0"
```